app/scripts/proactive/rest/studio-client.js

```diff
@@ -239,16 +258,8 @@ define(
 
                 });
             } else {
-                that.send_multipart_request(config.restApiUrl + "/submit", jobXml, {"sessionid": localStorage['pa.session']}, function (result) {
-                    if (result.errorMessage) {
-                        that.alert("Cannot submit the job", result.errorMessage, 'error');
-                    } else if (result.id) {
-                        that.alert("Job submitted", "<html></html><a href='/scheduler' target='_blank'>'" + result.readableName + "' submitted successfully (Id " + result.id +")</a></html>", 'success');
-                        that.setVisualization(result.id, visualization);
-                    } else {
-                        that.alert("Job submission", request.responseText, 'error');
-                    }
-                }, true);
+		console.log("normal submit");
+		that.alert("Cannot upload a file", "Cannot planned a workflow without EXECUTION_CALENDAR", 'error');
```

Comment:
instead of 
"Cannot planned a workflow without EXECUTION_CALENDAR"
better
"Cannot plan a workflow without EXECUTION_CALENDAR"